helm/vitess/values.yaml

```diff
@@ -33,43 +33,43 @@ config:
 
     # choose a backup service - valid values are gcs/s3
     # TODO: add file and ceph support
-    # backup_storage_implementation: gcs
```

Comment:
This is going to change the default behavior to be gcs by default. I would prefer if by default we just didn't configure backups.